crates/router/src/core/errors/api_error_response.rs

```diff
@@ -262,6 +264,7 @@ impl actix_web::ResponseError for ApiErrorResponse {
             | Self::VerificationFailed { .. }
             | Self::PaymentUnexpectedState { .. }
             | Self::MandateValidationFailed { .. }
+            | Self::DisputeFailed { .. }
```

Comment:
why are we marking `DisputeFailed` as `BAD_REQUEST`?

Reply:
Not sure :/